corehq/apps/app_manager/app_translations/generators.py

```diff
@@ -66,7 +66,7 @@ def _get_labels_to_comments(self):
 
             for form in module['forms']:
                 for question in form['questions']:
-                    if question['comment']:
+                    if question['comment'] and 'label_ref' in question:
                         form_dict[question['label_ref']].append(question['comment'])
```

Comment:
Isn't a question's label_ref unique?

Reply:
It should be yes.

Not clear about the intend of the question,
the check is to ensure `label_ref` is present in the question dict.

Reply:
I was asking because this stored a list of comments per question, but I didn't think it was ever possible to have more than one, so it seemed like a list was unecessary.  That point is moot after your last commit though.